src/js/pages/Search/index.js

```diff
@@ -20,8 +21,58 @@ export default class Search extends React.PureComponent {
     writer: PropTypes.string,
   };
 
+  state = {
+    searchURL: '',
+    verseIdList: [],
+    semanticInfo: {}
+  };
+
+  setSearchUrl() {
+    const { q, type, source, offset, writer } = this.props;
+    const isChatBot = type === SEARCH_TYPES.ASK_A_QUESTION;
+    let semanticInfo;
+
+    if (isChatBot) {
+      const semanticApi = encodeURI(`${GURBANIBOT_URL}search/?query=${q}`);
+      const semanticReq = fetch(semanticApi).then((response) => response.json());
+      semanticReq.then((semanticData) => {
+        const shabadIdList = semanticData.results.map((dataObj) => {
+          const { ShabadID, VerseID } = dataObj.Payload;
+          this.state.verseIdList.push(VerseID);
+          return ShabadID;
+        });
+
+        semanticInfo = {
+          pageResults: shabadIdList.length,
+          TotalResults: shabadIdList.length,
+          pages: {
+            page: 1,
+            resultsPerPage: 20,
```

Comment:
Does it make sense to add this to some config file or you think it's better suited here?

Reply:
This will come from banidb API, we won't need to write this manually here, its just a temporary code, until we get that endpoint in production.